lux/core/series.py

```diff
@@ -14,20 +14,26 @@
 
 import pandas as pd
 class LuxSeries(pd.Series):
-	# _metadata =  ['name','_intent','data_type_lookup','data_type',
-	# 			 'data_model_lookup','data_model','unique_values','cardinality',
-	# 			'min_max','plot_config', '_current_vis','_widget', '_recommendation']
+	_metadata = ['_intent','data_type_lookup','data_type',
+				 'data_model_lookup','data_model','unique_values','cardinality','_rec_info', '_pandas_only',
+				'_min_max','plot_config', '_current_vis','_widget', '_recommendation','_prev','_history', '_saved_export']
 	def __init__(self,*args, **kw):
 		super(LuxSeries, self).__init__(*args, **kw)
+
 	@property
 	def _constructor(self):
 		return LuxSeries
 
 	@property
 	def _constructor_expanddim(self):
 		from lux.core.frame import LuxDataFrame
-		# def f(*args, **kwargs):
-		# 	# adapted from https://github.com/pandas-dev/pandas/issues/13208#issuecomment-326556232
-		# 	return LuxDataFrame(*args, **kwargs).__finalize__(self, method='inherit')
-		# return f
+	# 	def f(*args, **kwargs):
```

Comment:
Is there a reason that we need to leave this commented? If so, it would be good to have another comment explaining why above this section. Or a TODO tag if deferring to a later date.

Reply:
Removed the comments, since we won't be needing to using `__finalize__` anymore.